*** keyboard.xml	2007-07-09 14:56:14.000000000 +0200
--- /usr/local/src/fgfs-builder-20070514/src/FlightGear_base/keyboard.xml	2007-07-09 15:00:25.000000000 +0200
*************** calculated by adding 256 to the GLUT key
*** 241,274 ****
   </key>
  
   <key n="32">
!    <name>SPACE</name>
!    <desc>PTT Key for COM</desc>
     <binding>
!        <command>property-assign</command>
!        <property>/instrumentation/comm/ptt</property>
!        <value type="int">1</value>
!    </binding>
!    <mod-up>
!      <binding>
!        <command>property-assign</command>
!        <property>/instrumentation/comm/ptt</property>
!        <value type="int">0</value>
!      </binding>
!    </mod-up>
!    <mod-shift>
!      <binding>
!        <command>property-assign</command>
!        <property>/instrumentation/comm/ptt</property>
!        <value type="int">2</value>
!      </binding>
!     <mod-up>
!      <binding>
!        <command>property-assign</command>
!        <property>/instrumentation/comm/ptt</property>
!        <value type="int">0</value>
!      </binding>
!    </mod-up>
!    </mod-shift>
   </key>
  
   <key n="33">
--- 241,264 ----
   </key>
  
   <key n="32">
!   <name>SPACE</name>
!   <desc>Fire Starter on Selected Engine(s)</desc>
!   <binding>
!    <command>nasal</command>
!    <script>controls.startEngine()</script>
!   </binding>
!   <mod-up>
!    <binding>
!     <command>nasal</command>
!     <script>props.setAll("/controls/engines/engine", "starter", 0)</script>
!    </binding>
!   </mod-up>
!   <mod-shift>
     <binding>
!     <command>nasal</command>
!     <script>gui.property_browser()</script>
!    </binding>
!   </mod-shift>
   </key>
  
   <key n="33">
*************** calculated by adding 256 to the GLUT key
*** 629,657 ****
    </binding>
   </key>
  
-  <key n="83">
-   <name>S</name>
-   <desc>Swap panels.</desc>
-   <binding>
-    <condition>
-      <not>
-      <property>/sim/allow-toggle-cockpit</property>
-      </not>
-    </condition>
-    <command>property-swap</command>
-    <property>/sim/panel/path</property>
-    <property>/sim/panel_2/path</property>
-   </binding>
-   <binding>
-    <condition>
-      <not>
-      <property>/sim/allow-toggle-cockpit</property>
-      </not>
-    </condition>
-    <command>panel-load</command>
-   </binding>
-  </key>
- 
   <key n="84">
    <name>T</name>
    <desc>Decrease warp delta.</desc>
--- 619,624 ----
*************** calculated by adding 256 to the GLUT key
*** 889,905 ****
  
   <key n="115">
    <name>s</name>
!   <desc>Fire Starter on Selected Engine(s)</desc>
    <binding>
!    <command>nasal</command>
!    <script>controls.startEngine()</script>
    </binding>
-   <mod-up>
-    <binding>
-     <command>nasal</command>
-     <script>props.setAll("/controls/engines/engine", "starter", 0)</script>
-    </binding>
-   </mod-up>
   </key>
  
   <key n="116">
--- 856,880 ----
  
   <key n="115">
    <name>s</name>
!   <desc>Swap panels.</desc>
    <binding>
!    <condition>
!      <not>
!      <property>/sim/allow-toggle-cockpit</property>
!      </not>
!    </condition>
!    <command>property-swap</command>
!    <property>/sim/panel/path</property>
!    <property>/sim/panel_2/path</property>
!   </binding>
!   <binding>
!    <condition>
!      <not>
!      <property>/sim/allow-toggle-cockpit</property>
!      </not>
!    </condition>
!    <command>panel-load</command>
    </binding>
   </key>
  
   <key n="116">
